jupyter_server/gateway/gateway_client.py

```diff
@@ -341,6 +341,24 @@ def gateway_retry_max_default(self):
             os.environ.get("JUPYTER_GATEWAY_RETRY_MAX", self.gateway_retry_max_default_value)
         )
 
+    kernel_launch_timeout_pad_default_value = 0
```

Comment:
I think it would be good to have a non-zero default.   This will serve as a better example and should (if properly defined) prevent an additional configuration step a majority of the time.  We previously used `2` which seems like a reasonable value to me.

Reply:
hmm - should we increase the `request_timeout` default to 42 or decrease KLT to 38 in this case?  Or just let it always adjust request_timeout to 42 in the default case.   I'm kinda leaning toward increasing the `request_timeout` default to 42 (since it's the answer to the universe anyway :smile: ).

Reply:
I'm also leaning towards increasing the `request_timeout`.